src/snowflake/snowpark/_internal/data_source/utils.py

```diff
@@ -146,7 +146,7 @@ def _upload_and_copy_into_table(
     )
     copy_into_table_query = f"""
     COPY INTO {snowflake_table_name} FROM @{snowflake_stage_name}/{file_name}
-    FILE_FORMAT = (TYPE = PARQUET)
+    FILE_FORMAT = (TYPE = PARQUET USE_VECTORIZED_SCANNER=TRUE)
```

Comment:
I'm checking the doc: https://docs.snowflake.com/en/sql-reference/sql/copy-into-table and there are some restrictions about this `USE_VECTORIZED_SCANNER`.

does any of the restrictions mentioned in the doc would impact our feature?